patches/ava+5.1.1.patch

```diff
@@ -6,3 +6,50 @@ index 0000000..bee62d8
 @@ -0,0 +1,2 @@
 +// XXX work around https://github.com/import-js/eslint-plugin-import/issues/1810
 +export {default} from './lib/worker/main.cjs';
+diff --git a/node_modules/ava/lib/cli.js b/node_modules/ava/lib/cli.js
+index afe5528..67f523f 100644
+--- a/node_modules/ava/lib/cli.js
++++ b/node_modules/ava/lib/cli.js
+@@ -436,6 +436,10 @@ export default async function loadCli() { // eslint-disable-line complexity
+ 		workerArgv: argv['--'],
+ 	});
+ 
++	if (process.env.AGORIC_AVA_USE_TAP){
++		combined.tap = true;
++	}
```

Comment:
I'd prefer to have this higher up and avoid mutating `combined` after it has already been consumed.